pom.xml

```diff
@@ -189,9 +183,18 @@
             </plugin>
         </plugins>
         <resources>
+            <resource>
+                <directory>${project.basedir}/src/main/resources</directory>
+            </resource>
```

Comment:
Why do you need to include the resources dir?

Reply:
Good question. This block isn't additive, but rather replaces the default resources directory for the project. Since my intention is not to replace the default resources directory, I redefined it here.

That being said, you'll notice we don't actually have any resources or a `src/main/resources` directory in this project, so this really isn't necessary. However, I decided to include it anyway to avoid confusion why resources aren't being picked up if we ever did need to add any in the future.

Reply:
I don't understand the reason for this, isn't it included by default
https://maven.apache.org/plugins/maven-resources-plugin/examples/resource-directory.html
?

Reply:
src/main/resources is included by default. I would remove it here.

Reply:
Yes, it's the default, but once you define anything else in the `resources` block then `src/main/resources` is no longer included. In other words, with the following configuration:

```xml
<resources>
    <resource>
        <directory>${project.basedir}/foo</directory>
    </resource>
</resources>
```

Will mean `src/main/resources` is no longer included as a resources directory. That's why I redefine it here.

Reply:
Oh, I see. In that case, given this unexpected behavior, I would add it, as you have it. And doing the same for the other maven extension where we added the license files as part of these updates.